plugins/in_tail/tail_fs_stat.c

```diff
@@ -116,6 +116,26 @@ static int tail_fs_check(struct flb_input_instance *ins,
             continue;
         }
 
+#ifdef FLB_SYSTEM_WINDOWS
+        HANDLE h;
+        FILE_STANDARD_INFO info;
+
+        h = _get_osfhandle(file->fd);
```

Comment:
should not we close this handle as it is defined as void*

Reply:
@sachinmsft  According to the documentation:

> To close a file whose operating system (OS) file handle is obtained by _get_osfhandle, call _close on the file descriptor fd. Never call CloseHandle on the return value of this function.
> 
> https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/get-osfhandle?view=vs-2019

So I thought it is not supposed to be closed. Am I reading it wrongly?

Reply:
My bad. I did not read the remarks and thought it as a normal file handle. 
no need to close it.

Reply:
Just OK. Now I resolved this discussion.